### PR TITLE
[new] Add coolprop

### PR DESCRIPTION
**Pull request by doomiris**
Pull request for issue: #13317

## Profile of the lib

* Git repository (required): https://github.com/CoolProp/CoolProp.git
* Official website (optional, not the repository): http://coolprop.sourceforge.net/
* NPM package url (optional): none
* License and its reference: MIT
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 47
  * Count of stars: 236
  * Count of forks: 115
* NPM download stats (optional): none
  * Downloads in the last day:
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [ ] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [ ] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [ ] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [ ] The parent of the commit(s) in the PR is not older than 3 days.
* [ ] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by MattIPv4** [comment]
Resolves #13317

**Comment by doomiris** [comment]
@dakshshah96 @sufuf3 @sashberd @extend1994 , dear admins, would you please help review PR [#13318](https://github.com/cdnjs/cdnjs/pull/13318) ?

**Comment by PeterDaveHello** [comment]
@doomiris I'll take a look.

File: ajax/libs/coolprop/package.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  "keywords": [
+    "coolprop",
+    "refprop",
+    "thermophysical"
+  ],
+  "author": "ibell, jowr, wandadars, etc.",
+  "license": "MIT",
+  "homepage": "https://github.com/CoolProp/CoolProp/",
+  "filename": "coolprop.min.js"
+}
```
Comment (PeterDaveHello) [inline ajax/libs/coolprop/package.json]: Missing final newline char at EOF.

**Comment by doomiris** [comment]
I can't rebase this branch, because network reason. some one help me... 
bot review says:  "ERROR. The branch coolprop for this pull request is a little bit old, please rebase this branch with the latest master branch from upstream!"

**Comment by doomiris** [comment]
i' had to create new PR #13479 for this. cause I can't rebase from master branch.

**Pull request closed without merging**